package.json

```diff
@@ -3,42 +3,44 @@
   "private": true,
   "version": "3.5.7",
   "description": "🐻 Bear necessities for state management in React",
-  "main": "index.js",
-  "module": "esm/index.js",
-  "types": "index.d.ts",
+  "main": "./index.js",
+  "module": "./esm/index.mjs",
+  "type": "commonjs",
+  "types": "./index.d.ts",
   "files": [
     "**"
   ],
   "exports": {
     "./package.json": "./package.json",
+    "./": "./",
```

Comment:
just curious, which bundler or something requires this?

Reply:
It's a fallback for the default `zustand/<package>` cjs exports, in case the conditional exports are not working but normal single line exports are supported by the bundler. 

Not specific to any bundler

Reply:
Okay. I didn't know about a bundler that may understand "exports" but not condtional exports.

Reply:
Same, saw 
https://github.com/marvinhagemeister/smoldash/pull/10/commits/38ea2735df076e1c60c7e2f8ddacbf46f0ed30ed
do it in the morning and looked up other repos that were doing it and based on a few conversations here and there , people have had issues for libraries with multiple entry points

Ex: I have [one](https://github.com/barelyhuman/wrap) which doesn't fully support conditional exports

Reply:
Can remove it, since we are manually declaring every single entry point anyway.

Reply:
If it helps, let's add it. (and make sure we add it to all zustand/jotai/valtio.)